web/styles/popovers.css

```diff
@@ -568,37 +568,27 @@ ul {
 
     .user_profile_manage_own_edit_button,
     .user_profile_manage_others_edit_button {
-        width: 25px;
-        height: 14px;
         text-align: center;
```

Comment:
Seems like we no longer need `text-align` property.

Reply:
Done